httpie/output/formatters/xml.py

```diff
@@ -8,13 +8,27 @@
     from xml.dom.minidom import Document
 
 
+XML_OPEN_TAG = '<?xml'
+XML_CLOSE_TAG = '?>'
```

Comment:
Pedantic: Would be nice to `DECL` in the name, and not have `TAG` in there as it can be confused with elements tags.